app/services/publish_manual_service.rb

```diff
@@ -8,6 +10,8 @@ def initialize(dependencies)
 
   def call
     if versions_match?
+      tags
```

Comment:
I'm going to answer the above questions out of order, because I think it might help to give more context.

> Are there any additional potential problems with this dependency between publishing and tag-fetching?

In the short-term, it's a problem because it slows down moving to the new publishing workflow. This discussion is documented in the [meeting notes](https://gov-uk.atlassian.net/wiki/display/FS/HMRC+manuals+update+-+29+JUL+2015). This PR unblocks a few things around manuals and collections. Removing this dependency will become something owned by the Finding Things team.

Additionally, it slows down the Publishing Platform team. They're trying to reduce the dependency that many of our publishing applications have on the [Content API](http://github.com/alphagov/govuk_content_api). The wider aim is to move everything over to use the [Content Store](https://github.com/alphagov/content-store) to produce a cleaner publishing workflow.

> Could there be an advantage to persisting a manual with its new tags regardless of the outcome of the publishing steps?

At the moment, the tagging is done in another application. Persisting a manual with its new tags for draft content would only be useful in terms of auditing what the tags for a given piece of content looked like when the draft was saved. As far as I'm aware, this isn't a user need at this time.

There are currently discussions going on about what this tagging implementation might look like and how it interacts with workflow and auditing. It's in the very early stages and this might be better there.

> Are there situations where we'd currently (and/or in the near future) still be able to (and want to) publish while some services are down? (If you see what I mean..?)

I think we want our publishing platform (in the future) to gracefully degrade where possible. However in the short term we have to make this step. It will help us update the data that we hold for manuals in [Rummager](http://github.com/alphagov/rummager) and their related tags. Once we have that in place we can start to use Rummager as one of the sources of truth for tagging content items and pull apart this dependency.

Let me know if there's anything I've missed.